src/main/kotlin/frc/team5499/frc2018Kotlin/Constants.kt

```diff
@@ -63,6 +63,24 @@ object Constants {
         const val INVERT_TURN_AUX_PIDF = false
     }
 
+    object Arm {
+        const val INTAKE_SPEED: Double = -0.7 
+        const val INTAKE_HOLD_SPEED: Double = -0.3
+        const val INTAKE_SPIT_SPEED: Double = 0.7
+        const val MAX_ARM_SPEED: Double = 0.7  
+        const val ARM_PORT: Int = 5
+        const val LEFT_INTAKE_PORT: Int = 4
+        const val RIGHT_INTAKE_PORT: Int = 6
+
+        const val INVERT_ARM: Boolean = false
+        const val INVERT_INTAKE: Boolean = false
+    }
+
+    object Controller {
+        const val XBOX_DEADBAND: Double = 0.1
+        const val XBOX_TURN_MULTIPLIER: Double = 0.4
```

Comment:
I think there's an "Input" object in constants, they should go there if there is. Double check pls